packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadiness.ts

```diff
@@ -290,7 +293,6 @@ export function useTransferReadiness({
     // The amount entered is enough funds, but now let's include gas costs
     switch (gasSummary.status) {
       // No error while loading gas costs
-      case 'idle':
```

Comment:
what happened to `idle`? is it no longer necessary?

Reply:
no longer needed because we always load it